apps/web/app/layouts/NavigationLayout.tsx

```diff
@@ -0,0 +1,26 @@
+"use client"
+
+import styled from "styled-components";
+import Nav from "../Nav";
+import SideNav from "../SideNavigationBar";
+import { LayoutProps } from "../layout";
+
+const Main = styled.div`
```

Comment:
Since it is ```Main``` I would change tag to ```styled.main```
```suggestion
const Main = styled.main`
```

Reply:
That's a really good idea, I will change it!